publiq-ui/pub-radio-group.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <b-form-group :label="groupLabel" class="form-group">
+    <b-form-radio-group v-model="selectedModel" stacked>
+      <b-form-radio
+        v-for="(value, index) in values"
+        :key="value"
+        :value="value"
+        :name="name"
+        class="custom-radio"
+        >{{ labels[index] }}</b-form-radio
+      >
```

Comment:
This formatting looks really weird. Was this suggested by the linter? 🤔

Reply:
It's automatically formatted by prettier. It seems to be correct since the prettier check on Github Actions is green for this PR.